exercises/takudzwa_makoni/ex02_oo_basics/src/main.cpp

```diff
@@ -0,0 +1,46 @@
+
+//#include "pch.h"
+
+#include <iostream>
+#include <math.h>
+#include <vector>
+#include <string>
+#include <algorithm>
+#include "shape.h"
+#include "square.h"
+#include "rectangle.h"
+#include "isoTriangle.h"
+#include "circle.h"
+#include "shapeSorter.h"
+
+
+square s; circle c; rectangle r; isoTriangle t;
+std::vector<shape*> shapeVector({ &s, &c, &r, &t });
```

Comment:
Already discussed `std::vector<unique_ptr<shape>>`. Also should not be in defined global scope.